Sources/Overload/OvUI/src/OvUI/Core/UIManager.cpp

```diff
@@ -227,11 +235,11 @@ bool OvUI::Core::UIManager::IsEditorLayoutSaveEnabled() const
 	return ImGui::GetIO().IniFilename != nullptr;
 }
 
-void OvUI::Core::UIManager::SetEditorLayoutSaveFilename(const std::string & p_filename)
+void OvUI::Core::UIManager::SetEditorLayoutSaveFilename(const std::filesystem::path& p_filePath)
 {
-	m_layoutSaveFilename = p_filename;
+	m_layoutSaveFilename = p_filePath;
 	if (IsEditorLayoutSaveEnabled())
-		ImGui::GetIO().IniFilename = m_layoutSaveFilename.c_str();
+		ImGui::GetIO().IniFilename = m_layoutSaveFilename.string().c_str();
```

Comment:
This introduces a bug where the editor layout doesn't get saved anymore when quitting.
This is due to the return value of `m_layoutSaveFilename.string().c_str();` becoming a dangling pointer, since the generated `std::string` with `string()` is destroyed on the next line, while ImGui requires this string to stay alive until the editor closes

Reply:
`ImGui::GetIO().IniFilename` is non owning, so we need to have an allocated string somewhere in memory